### PR TITLE
docs: add requirements for PR labels in contribution guide

### PR DESCRIPTION
**Pull request by Leoyoungxh**
In order to generate the changlog quickly during the release process, PR guidelines needs to be further improved.

Improve the PR template and documentation, requiring the inclusion of release notes in the PR description and setting appropriate labels.

RELEASE NOTES: NONE

### CODE REVIEW

**Comment by Leoyoungxh** [comment]
In the future, mayby consider using the [mergeable](https://github.com/mergeability/mergeable) plugin to check if the PR description meets the requirements.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ Here is an example of a good one:
 > The algorithm is described at https://wikipedia.org/wiki/McGillicutty_Algorithm
 >
 > Fixes #159
-
+>
```
Comment (liuzengh) [inline CONTRIBUTING.md:78]: Should the PR type label appear in the commit message? If so, could you provide an example here?

Comment (Leoyoungxh) [reply CONTRIBUTING.md:78]: I don't think PR type lable should appear in the commit message, it's duplicated.
Contributors just need to set label in the PR website.
image

**Comment by liuzengh** [comment]
Perhaps we can create a release guideline document first, to understand the overall process of releasing, and then add the relevant information to the contributor guide

**Comment by Leoyoungxh** [comment]
> Perhaps we can create a release guideline document first, to understand the overall process of releasing, and then add the relevant information to the contributor guide

Sounds good! But I think we can add the release guideline later. Compared to contributors, repository maintainers are more concerned about the release process.

**Review by liuzengh — APPROVED** [review]
LGTM

**Review by sandyskies — APPROVED** [review]
LGTM

**Pull request merged**